genesis/genesis.go

```diff
@@ -355,13 +356,14 @@ func AVAXAssetID(avmGenesisBytes []byte) (ids.ID, error) {
 		c.RegisterType(&secp256k1fx.TransferOutput{}),
 		c.RegisterType(&secp256k1fx.MintOperation{}),
 		c.RegisterType(&secp256k1fx.Credential{}),
+		m.RegisterCodec(0, c),
```

Comment:
Let's use a variable (`codecVersion` or something) instead of magic number `0` here and on lines 374 and 378